### ISSUE TITLE
Edge probe 

### ISSUE THREAD

**Issue by jratha** [gnea/grbl#538]
hi all

I’m after a bit of help. I’ve made a combined edge probe and touch plate for my cnc. Connected the positive to the plate and negative to my spindle. The positive is connected to port a5 on my Arduino Uno and ground to the ground pin. But no matter what I do, I can’t seam to get the probe to work, is there something I’ve missed as apart from how to use the probe, I can’t seam to find any tutorials on how to set it up.

I’ve checked the continuity on the wiring and all if fine, but just won’t work then the tool touches the plate

Hope someone can help with this

**Comment by zeevy** [comment]
You said you attached the negative to spindle!, it should be connected to the tool since in most of DC spindles tool bit is isolated from the body of the motor.

You can check your setup by simply touching the both probing  wires  after a probe cycle begins.

**Comment by jratha** [comment]
Ok so it looks like I’ve managed to clear up the probe. Tried you suggestion of touching the leads together after initianating and didn’t work.  Tried other ground ports on Arduino and still nothing. In a last attempt I connected the ground to the limit switch ground with pull up resistors and it now appears to work, sort of.

`G38.2 Z-40 F60;
G92 Z5.80;
G91;
G0 Z5;
M30


G38.2 X-50 F60;
G92 X3.175;
G91;
G0 X10;
M30

G38.2 Y50 F60;
G92 Y3.175;
G91;
G0 Y10;
M30`

This the macros I’ve programmed in to UGS. All are in mm. the y and z probe appear to work ok, but when I start the x probe, it starts the routine but immediately retracts. It’s the exact same code but won’t work on x axis
